opensource/doc-style.md

```diff
@@ -221,14 +221,14 @@ When writing CLI examples, give the user hints by making the examples resemble
 exactly what they see in their shell:
 
 * Indent shell examples by 4 spaces so they get rendered as code blocks.
-* Start typed commands with `$ ` (dollar space), so that they are easily
+* Start typed commands with <tt>$ </tt> (dollar space), so that they are easily
   differentiated from program output.
 * Program output has no prefix.
 * Comments begin with # (hash space).
-* In-container shell commands, begin with `$$ ` (dollar dollar space).
+* In-container shell commands, begin with <tt>$$ </tt> (dollar dollar space).
```

Comment:
http://i.imgur.com/bdG4KEF.png  <-- this is what you get with `<tt>` tags, which is not desirable